modules/dxp/apps/commerce/commerce-organization-web/src/main/java/com/liferay/commerce/organization/web/internal/display/context/CommerceOrganizationDisplayContext.java

```diff
@@ -41,10 +42,12 @@ public class CommerceOrganizationDisplayContext {
 
 	public CommerceOrganizationDisplayContext(
 			HttpServletRequest httpServletRequest,
+			OrganizationLocalService organizationLocalService,
 			OrganizationService organizationService,
 			UserLocalService userLocalService)
 		throws PortalException {
 
+		_organizationLocalService = organizationLocalService;
```

Comment:
Remove `_themeDisplay` variable and use `_commerceOrganizationRequestHelper` instead.

Also, pass ConfigurationProvider to constructor instead.

Reply:
Same for PortalUtil, pass Portal to constructor instead.